provisioner/clusterpy.go

```diff
@@ -1137,6 +1146,38 @@ func performDeletion(ctx context.Context, logger *log.Entry, client dynamic.Inte
 	return nil
 }
 
+func annotateDeletion(ctx context.Context, kind, name string, iface dynamic.ResourceInterface) error {
```

Comment:
Let's rename this so it's easier to understand what this is about. It's some sort of deletion protection, right? So maybe `disableDeletionProtection` or something like that? (similar to CloudFormation's Stack deletion protection)

Reply:
@myaser Thanks!